reactivexcomponent/reactivexcomponent/communication/publisher.py

```diff
@@ -0,0 +1,96 @@
+""" Publisher """
+
+import json
+import websocket
+from reactivexcomponent.configuration.serializer import *
```

Comment:
better avoid import *

http://stackoverflow.com/questions/2386714/why-is-import-bad